core/src/smartcontracts/isi/triggers/mod.rs

```diff
@@ -6,7 +6,7 @@ use iroha_data_model::{
 };
 use iroha_telemetry::metrics;
 
-pub(crate) mod set;
```

Comment:
Could be made more granular, but almost the whole module leaks through with `WorldStateView::triggers` anyways, so I decided against it unless someone objects.

Reply:
That was the original intention. `WorldStateView::triggers` is inspired by SBCL hooks.